fhir/group.js

```diff
@@ -33,7 +33,7 @@ function resourceCreator(multiplier) {
     }
 }
 
-function bundle(items, multiplier) {
+function bundle(req, items, multiplier) {
```

Comment:
Same comment about only passing in baseUrl

Reply:
made change